pkg/grapicmd/di/providers.go

```diff
@@ -55,7 +55,7 @@ func ProvideGenerator(ctx *grapicmd.Ctx, ui clui.UI) module.Generator {
 	)
 }
 
-func ProvideScriptLoader(ctx *grapicmd.Ctx, executor command.Executor) module.ScriptLoader {
+func ProvideScriptLoader(ctx *grapicmd.Ctx, executor excmd.Executor) module.ScriptLoader {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function ProvideScriptLoader should have comment or be unexported